TTHAnalysis/python/plotter/ttH-multilepton/2lss_3l_plots_diff.txt

```diff
@@ -1,13 +1,27 @@
 # vim: syntax=sh
 2lep_2DpT_2bins: Hreco_pTHvis\:Hreco_pTHgen :  [0,120,350]*[0,120,350]         ;                  XTitle="pTHvis (GeV)",  YTitle="pTHgen (GeV)" , Legend='TL', LegendCutoff=1e-5, PlotMode='COLZ TEXT', IncludeOverflows=True
+#2lep_2DpT_2bins_withNu: Hreco_pTVisPlusNu\:Hreco_pTHgen :  [0,120,350]*[0,120,350]         ;                  XTitle="pTHvisplusNu (GeV)",  YTitle="pTHgen (GeV)" , Legend='TL', LegendCutoff=1e-5, PlotMode='COLZ TEXT', IncludeOverflows=True
```

Comment:
remove commented lines addition and empty line addition

Reply:
empty lines removed. But, commented lines kept for later purposes.